spec/factories/user.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :user do
+    email     { 'test_user@example.com' }
+    password  { Faker::Internet.password }
+    full_name { Faker::Name.name_with_middle }
+    gender    { 2 }
```

Comment:
You can use something like: `User.genders.keys.sample` (.sample returns a random element from the array)